### ISSUE TITLE
Teleport Death

### ISSUE THREAD

**Issue by LoXR** [EssentialsX/Essentials#2110]
### Information

Full output of `/ess version`: Essentials version 2.15.0.17

Server log (upload `logs/latest.log` to [Gist](https://gist.github.com/)): https://gist.github.com/LoXR/290236a8fbe557ffbc3a5b9418f7778e

EssentialsX config (upload `plugins/Essentials/config.yml` to [Gist](https://gist.github.com/)): https://gist.github.com/LoXR/b7bb8606ca66ad389a777566f533838d

### Details

Describe your issue here.
Basically when players go /home they are teleported and upon landing at home, they are killed with the message "player fell from a high place" all though they just teleported and all homes are on a flat surface.

**Comment by mdcfe** [comment]
Please include the full output of `/essentials version` as asked, **not just the Essentials version**.

**Comment by LoXR** [comment]
Server version: 1.12.2-R0.1-SNAPSHOT git-Spigot-eb3d921-b9f1615 (MC: 1.12.2)
EssentialsX version: 2.15.0.17
Vault version: 1.6.1
EssentialsXChat version: 2.15.0.17
EssentialsXSpawn version: 2.15.0.17

**Comment by mdcfe** [comment]
* Does this happen with every home?
* Does this happen if you create a warp in the exact same location as the home and teleport to that warp?

**Comment by LoXR** [comment]
It seems to only happen a few times. Havent tried that as its only occured to set homes.

**Comment by mdcfe** [comment]
Please try and replicate this and respond with any results.

**Comment by LoXR** [comment]
Havent heard any complaints from players, so will say its somehow sorted itself out.
